src/components/ControlPanel/ControlPanelBox/ControlPanelBox.js

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
 import * as S from './ControlPanelBox.styles';
+import { useIntl } from 'react-intl';
 
 export const ControlPanelBox = ({ box }) => {
+  const _ = (id, values) => useIntl().formatMessage({ id }, values);
+
   return (
     <div className="col-lg-3 col-md-4 col-sm-4 m-b-24">
       <div className="dp-box-shadow">
-        <S.Link target="_self" className="dp-white" href={box.linkUrl}>
-          <S.Image src={box.imgSrc} alt={box.imgAlt} />
-          <S.Text>{box.iconName}</S.Text>
+        <S.Link target="_self" className="dp-white" href={_(box.linkUrl)}>
```

Comment:
Ahh, ahora entiendo!

Creo que la URL debería definirse directamente en los `linkUrl` de los `items` de `controlPanelSections`, no es necesario traducirlas.

```suggestion
        <S.Link target="_self" className="dp-white" href={box.linkUrl}>
```